components/logins/src/sync/payload.rs

```diff
@@ -16,8 +16,60 @@ use serde_derive::*;
 use sync15::bso::OutgoingBso;
 use sync_guid::Guid;
 
+pub(super) type UnknownFields = serde_json::Map<String, serde_json::Value>;
+
+/// What we get from the server after parsing the payload. We need to round-trip "unknown"
+/// fields, but don't want to carry them around in `EncryptedLogin`.
+#[derive(Debug)]
+pub(super) struct IncomingLogin {
+    pub login: EncryptedLogin,
+    pub unknown: UnknownFields,
+}
```

Comment:
Maybe down the line this could be made generic/shared.  Like having a `sync15::IncomingRecord<T>` type that wraps a concrete sync record adds an `unknown_fields`.